googlebot_site_map/views_admin.py

```diff
@@ -0,0 +1,22 @@
+# googlebot_site_map/views_admin.py
+# Brought to you by We Vote. Be good.
+# -*- coding: UTF-8 -*-
+import re
+
+import wevote_functions.admin
+from politician.models import Politician
+
+logger = wevote_functions.admin.get_logger(__name__)
+
+def get_googlebot_map_file_body(request):
+    print(request.META['HTTP_USER_AGENT'])
+    map_num_result = re.findall(r'googlebotSiteMap\/map(\d+)', request.path)
+    num = int(map_num_result[0])
+    https_root = "https://wevote.us/"
+    map_text = ''
```

Comment:
It would be helpful to have the regular paths (like "/ready", "/value/democratic_clubs" etc.) listed at the top of this file, so we can quality check the standard URLs before listing the 75000 politician URL after that.